### PR TITLE
[astro] Update README.md

### PR DESCRIPTION
**Pull request by uqs**
Help users detangle #start, #end and #event channels and their respective offsets. I was struggling with this and it took reading https://community.openhab.org/t/solved-open-rollershutters-based-on-astro-binding-with-offset-does-not-work/123336 to figure out where I went wrong.

While here, maybe fix the Markdown rendering of QuantityType<Intensity>

### CODE REVIEW

File: bundles/org.openhab.binding.astro/README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ astro:sun:home  [ geolocation="52.5200066,13.4049540,100", interval=60 ]
 astro:moon:home [ geolocation="52.5200066,13.4049540", interval=60 ]
 ```
 
-or optionally with an event offset
+or optionally with an event offset (this can be used in rules triggers)
```
Comment (lsiepel) [inline bundles/org.openhab.binding.astro/README.md:148]: This part is about the thing declaration. It has nothing to do with triggers itself. 
If you want to clarify how the event / trigger channel (with or without the offset) can be consumed, i would suggest to add some notes at line 97.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Thanks, for adding some clarification to the event channel. 
Not sure of DCO is a problem, would be nice if you can correct that too.

File: bundles/org.openhab.binding.astro/README.md (hunk 1 above)
Comment (uqs) [reply bundles/org.openhab.binding.astro/README.md:148]: I've reverted that part, it's spelled out on line 199

**Review by lsiepel — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.astro/README.md [hunk 2]
```diff
@@ -196,7 +196,7 @@ Number:Intensity Total_Radiation    "Radiation [%.2f %unit%]"                 {
 Number:Intensity Diffuse_Radiation  "Diffuse Radiation [%.2f %unit%]"         { channel="astro:sun:home:radiation#diffuse" }
 ```
 
-Events:
+Events (Note: offset/latest on #start or #end are independent from offset/latest on the #event, and only #event can be used in rules triggers like below):
```
Comment (lolodomo) [inline bundles/org.openhab.binding.astro/README.md:199]: "earliest" is not mentioned. Did you mismatch "offset" and "earliest" ?

"offset", "earliest" and "latest" are channel options I believe, so yes of course they can't be use as trigger in a rule.
Maybe you should rather add your explanations in the part of the README defining these parameters ?

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.astro/README.md (hunk 2 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.astro/README.md:199]: I would suggest to move it to line 97

Comment (uqs) [reply bundles/org.openhab.binding.astro/README.md:199]: > "earliest" is not mentioned. Did you mismatch "offset" and "earliest" ?

No, I simply didn't want to enumerate everything again.
 
> "offset", "earliest" and "latest" are channel options I believe, so yes of course they can't be use as trigger in a rule. Maybe you should rather add your explanations in the part of the README defining these parameters ?

You say "of course", but this is far from obvious for a normal user like me, I had to debug and research this for several evenings when the rules didn't trigger as I thought they should trigger, even though the sunset item was showing the offset correctly.

I've moved this up to line 97 as suggested.

**Review by lolodomo — APPROVED** [review]
Let's merge this changer even if the explanation about offset config is not so clear in my opinion.

**Pull request merged**